jest-preset.js

```diff
@@ -0,0 +1,15 @@
+// jest-preset.js
+
+// eslint-disable-next-line unicorn/prefer-module
+module.exports = {
+  preset: 'ts-jest',
+  collectCoverageFrom: ['<rootDir>/src/**', '!<rootDir>/src/**/*.spec.ts'],
```

Comment:
`'!<rootDir>/src/**/*.test.ts'`

Reply:
I eliminated that on purpose.  wallaby doesn't seem to work with those files, and I think unused .test.ts code is less than ideal.